FrontEnd/myapp/src/StockCheck.js

```diff
@@ -12,23 +12,24 @@ const apiKey = "RZEXR5SFIKLQNKYK";
 
 function StockCheck() {
   let myvalue, myquery;
-  const [price, setPrice] = useState(null);
-  const [ticker, setTicker] = useState("");
+  const [message, setMessage] = useState("");
 
   async function HandleSubmit(e) {
     console.log(e.target.ticker.value);
     e.preventDefault();
-    setTicker(e.target.ticker.value.toUpperCase());
-    //console.log(ticker)
+    const ticker = e.target.ticker.value.toUpperCase()
```

Comment:
Interesting change of taking the ticker out of the state, would it be more useful to still track it in state?